### ISSUE TITLE
Naming convention in twiss and survey tables 

### ISSUE THREAD

**Issue by sterbini** [MethodicalAcceleratorDesign/MAD-X#1020]
Hi,

after having opened the issue  https://github.com/hibtc/cpymad/issues/91 
we think that the proper forum of discussion is this one. 

We noticed that in cpymad '1.8.1' when we access the `twiss` and `survey` tables  we find different names for the same element.
E.g.:

```
>> mad.table.twiss[0].name
'ip3:1'
```
but
```
>> mad.table.survey[0].name
'ip3'
```

This seems due to a different strategy adopted in MAD-X to represent the elements names in the `twiss` and `survey` tables (the tables are handled in a identical way in `cpymad`).
Would it possible to use a common naming convention  for all MAD-X tables?
Thanks.

**Comment by tpersson** [comment]
I just discussed with Riccardo and this is coming from the fact that I was adding an ending to the name in case of permanent misalignment is chosen (so this is the last release). Internally the name is connected to the node so IP3:1 means that it is the first IP3 in the sequence. If you would output the survey table, there would be no difference, but when you use cpymad you get the table's internal structure, which is different. I will update so you also internally get the same name ip3:1

**Comment by tpersson** [comment]
Fixed in #1024. Back to the old behavior (same as in TWISS)
